README.md

```diff
@@ -0,0 +1,44 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Can we please change this README

Reply:
This will probably be addressed [here](https://github.com/OpenSRP/web/pull/12)

Reply:
should be addressed [here](https://github.com/OpenSRP/web/pull/12)